com.microsoft.mrtk.tools/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "com.microsoft.mrtk.tools",
+  "version": "3.0.0-development",
```

Comment:
This should be `1.0.0-development` if the package was not present before in any prior release of MRTK, right?

Reply:
we had a tools package in MRTK2, so i started this one at 3.0.0

Reply:
Had no idea! LOL